### ISSUE TITLE
`wizzy import dashboards` claims to start downloading but does not

### ISSUE THREAD

**Issue by sparr** [utkarshcmu/wizzy#117]
I can `wizzy import dashboard some-slug` successfully, but `wizzy import dashboards` says that it is `Importing 33 dashboards:` but then exits without further stdout or file output.

**Comment by sparr** [comment]
Complementary issue:

```% wizzy export dashboards
Output:
{"message":"Unauthorized"}
✘ Dashboard slug-1 export failed.
Output:
{"message":"Unauthorized"}
✘ Dashboard slug-2 export failed.
```

but `wizzy export dashboard slug-1` is successful

**Comment by DanCech** [comment]
Are you using latest master? This sounds like the issue fixed in #116

**Comment by sparr** [comment]
Good call. Sorry for the dupe.
